mod/photos.php

```diff
@@ -1035,14 +1035,10 @@ function photos_content(App $a)
 		if ($cmd === 'drop') {
 			$drop_url = $a->query_string;
 
-			$extra_inputs = [
-				['name' => 'albumname', 'value' => $_POST['albumname']],
-			];
-
 			return Renderer::replaceMacros(Renderer::getMarkupTemplate('confirm.tpl'), [
 				'$method' => 'post',
 				'$message' => L10n::t('Do you really want to delete this photo album and all its photos?'),
-				'$extra_inputs' => $extra_inputs,
+				'$extra_inputs' => [],
```

Comment:
What does now happen on deleting - or not deleting - the album?

Reply:
One less PHP Notice. This unused form input was created from non-existent data. I assume it's an artifact left from before the album guid was used in the drop action URL.

Reply:
Ah I see.